specs/engage-digital_openapi3.yaml

```diff
@@ -8242,6 +8265,25 @@ components:
           type: integer
         status:
           type: string
+        template_messages:
+          description: Only for WhatsApp Sources, list of the available message templates that can be used to send outbound messages.
+          items:
+            type: object
+            properties:
+              components:
+                items:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    text:
+                      type: string

```

Comment:
Nice catch ! as it's horrible to review, I'll spare you the headache and give you a screenshot of the rendered result  : 

<img width="689" alt="image" src="https://user-images.githubusercontent.com/29726012/176716155-5995d8fd-c693-4776-be0e-cb573f973117.png">

Reply:
thanks for the screenshot :+1: I guess we don't have any way of marking these fields as optional?

Reply:
FTR I found https://swagger.io/specification/#discriminator-object which would allow us to make a difference between WA sources and standard sources, using two different models.

But IMO it's not really needed.
We don't have a good way to mark those two "url" and "phone_number" as optional though